content/docs/command-reference/remote/add.md

```diff
@@ -322,7 +322,7 @@ like `ssh` and `sftp` (GNU/Linux).
 ### Click for HDFS
 
 ```dvc
-$ dvc remote add -d myremote hdfs://user@example.com/path/to/dir
+$ dvc remote add -d myremote hdfs://user@example.com/absolute/path
```

Comment:
we have abs in some places in some absolute - not a big deal. jfyi

Reply:
Yeah it was to keep it short but maybe "abs" is confusing. Anyway, will probably have to come up with another sample path per https://github.com/iterative/dvc.org/pull/1649#discussion_r468679544... ⏳

Reply:
Left only full "absolute" ones now.